manifests/image-references

```diff
@@ -0,0 +1,15 @@
+kind: ImageStream
```

Comment:
Should this not be a `.yaml` file? Also, did you manually add this or was it generated by `make generate`? If manually, we should probably just add this manifest to `manifests` and a copy target to `make generate`.

Reply:
https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/operators.md#how-do-i-ensure-the-right-images-get-used-by-my-manifests is the available documentation for this file. No yaml file extension is needed as it's a known downstream filename.